README.md

```diff
@@ -1,7 +1,293 @@
+        AVG(loss) OVER (ORDER BY date ROWS BETWEEN 13 PRECEDING AND CURRENT ROW) AS avg_loss
+    FROM gains_losses
+)
+SELECT 
+    date,
+    100 - (100 / (1 + avg_gain / avg_loss)) AS rsi
+FROM avg_gains_losses;
+```
+
+### History Point
+After the world stood still in 2020, Bitcoin was not spared, it went downhill to $6.9k, only to skyrocket from $34.7k to $60k in 2021 outdoing its previous year by $12.7k. 
+Then came the bear's grip in 2022 and just like gravity it went down, starting at $41.1k and ended quietly at $16.9k.
+In 2023, Bitcoin took a twist and climbed up starting at $20.2k and ended the year on a high note at $42.5k, due to this it had its highest amount of transaction at 731 in December 31, 2023.
+
+The bull's horn:- This beautiful ride that started in 2023 continued to the mountain top in 2024, the year price started at $42.8k and continues to climb up to $70.4k in March 2024. The Bitcoin halving that happens every 4 years is supposed to happen this year it was anticipated to take place on April 20, 2024. During this event, the block reward for mining Bitcoin will be reduced from 6.25 BTC to 3.125 BTC and we are all waiting for it to happen.
+
+How much Bitcoin do you think will drop out of the market, we are waiting to find out and do you think the market will go bearish or bullish?
+
+### Results
+The analysis results are summarized as follows:
+
+![whale picture](./whale_picture.PNG)
+![Price data](./Price_data.PNG)
+![btc total fees per date data](./btc_total_fees_per_date_data.PNG)
+![btc transaction data](./btc_transaction_data.PNG)
+
+#### Answers 
+- I guess there are not alot of people that can buy 100k worth of Bitcoin but the big whalers can.
+- The RSI showed that if its over 70, it was over bought, which indicates that the particular asset has been experiencing strong upward momentum, likewise if its below 30, it was over sold whcih indicates that the asset experienced a strong downward momenetum but in this case they maintained a balance as shown in the dasboard
+- I think Bitcoin price changes more on certain days of the week and over time, in 30 days it evens out and covers up for the days it messed up if you hold it for longer.
+- In the long term, Bitcoin is an uptrend if you are willing to be patient and ignore how volatile the market can get.
+
+### Recommendations
+In conclusion, Bitcoin's journey has been a rollercoaster ride full of highs and lows. 2024 has been a good high ride especially because we are all looking forward to the quaterly halving that happens every 4 years where half of the bitcoin gets burned out of the market, This halving is a significant event in the Bitcoin ecosystem as it reduces the rate at which new bitcoins are created, which historically has had a bullish impact on the price of Bitcoin​, i will advise you keep holding your Btc so you dont miss out on the massive bull that will take place when the halving happens.
```

Comment:
Line 4: the massive bull ^run that will take place

Reply:
Thank you, I have reviewd it